app/src/main/java/com/cornellappdev/android/eatery/WeeklyMenuFragment.java

```diff
@@ -0,0 +1,364 @@
+	private TextView date0;
+	private TextView date1;
+	private TextView date2;
+	private TextView date3;
+	private TextView date4;
+	private TextView date5;
+	private TextView date6;
+	private ExpandableListAdapter listAdapterWest;
+	private ExpandableListAdapter listAdapterNorth;
+	private ExpandableListAdapter listAdapterCentral;
+	private NonScrollExpandableListView expListViewWest;
+	private NonScrollExpandableListView expListViewNorth;
+	private NonScrollExpandableListView expListViewCentral;
+	private TextView westText;
+	private TextView northText;
+	private TextView centralText;
+	private MealType mealType = MealType.BREAKFAST;
+	private TextView breakfastText;
+	private TextView lunchText;
+	private TextView dinnerText;
+	private LinearLayout linDate;
+	private ArrayList<TextView> dateList = new ArrayList<>();
+	private WeeklyPresenter presenter;
+	private int lastExpandedPosition;
+	private NonScrollExpandableListView lastClickedListView;
+
+	public WeeklyMenuFragment() {
+		// Required empty public constructor
+	}
+
+
+	@Override
+	public View onCreateView(LayoutInflater inflater, ViewGroup container,
+	                         Bundle savedInstanceState) {
+		// Inflate the layout for this fragment
+		View rootView = inflater.inflate(R.layout.fragment_weekly_menu, container, false);
+		presenter = new WeeklyPresenter(rootView);
+		breakfastText = rootView.findViewById(R.id.breakfast);
+		lunchText = rootView.findViewById(R.id.lunch);
+		dinnerText = rootView.findViewById(R.id.dinner);
+		linDate = rootView.findViewById(R.id.lin_date);
+		expListViewWest = rootView.findViewById(R.id.expandablelistview_west);
+		expListViewNorth = rootView.findViewById(R.id.expandablelistview_north);
+		expListViewCentral = rootView.findViewById(R.id.expandablelistview_central);
+		westText = rootView.findViewById(R.id.west_header);
+		northText = rootView.findViewById(R.id.north_header);
+		centralText = rootView.findViewById(R.id.central_header);
+
+		// Set OnClickListener for dates
+		date0 = rootView.findViewById(R.id.date0);
+		date1 = rootView.findViewById(R.id.date1);
+		date2 = rootView.findViewById(R.id.date2);
+		date3 = rootView.findViewById(R.id.date3);
+		date4 = rootView.findViewById(R.id.date4);
+		date5 = rootView.findViewById(R.id.date5);
+		date6 = rootView.findViewById(R.id.date6);
+
+		date0.setOnClickListener(this);
+		date1.setOnClickListener(this);
+		date2.setOnClickListener(this);
+		date3.setOnClickListener(this);
+		date4.setOnClickListener(this);
+		date5.setOnClickListener(this);
+		date6.setOnClickListener(this);
+
+		// Layout for menu list
+		getActivity().setTitle("Upcoming Menus");
+		DisplayMetrics dm = new DisplayMetrics();
+		getActivity().getWindowManager().getDefaultDisplay().getMetrics(dm);
+		int width = dm.widthPixels;
+		expListViewWest.setIndicatorBounds(width - 250, width);
+		expListViewNorth.setIndicatorBounds(width - 250, width);
+		expListViewCentral.setIndicatorBounds(width - 250, width);
```

Comment:
I don't see the benefits of declaring `width` here, it takes a extra second to figure out that `width` is `dm.widthPixels`. I would just replace `width` with `dm.widthPixels`!

If anything, `250` should be denoted in a constant up top, because it's not clear where this number is from

Reply:
fixed! 250 is an arbitrary number, so I substituted that for an actual calculation of the offset